cmd/storagenode-updater/main.go

```diff
@@ -283,16 +330,26 @@ func unpackBinary(ctx context.Context, archive, target string) (err error) {
 	return nil
 }
 
-func restartSNService(name string) error {
+func restartService(name string) error {
 	switch runtime.GOOS {
 	case "windows":
-		// TODO how run this as one command `net stop servicename && net start servicename`?
 		// TODO: combine stdout with err if err
-		_, err := exec.Command("net", "stop", name).CombinedOutput()
+		restartSvcBatPath := filepath.Join(os.TempDir(), "restartservice.bat")
+		restartSvcBat, err := os.Create(restartSvcBatPath)
+		if err != nil {
+			return err
+		}
+
+		restartStr := fmt.Sprintf("net stop %s && net start %s", name, name)
+		_, err = restartSvcBat.WriteString(restartStr)
 		if err != nil {
 			return err
 		}
-		_, err = exec.Command("net", "start", name).CombinedOutput()
+		if err := restartSvcBat.Close(); err != nil {
+			return err
+		}
+
+		_, err = exec.Command(restartSvcBat.Name()).CombinedOutput()
```

Comment:
Can't we just use https://godoc.org/golang.org/x/sys/windows/svc/mgr instead of creating temp batch files?

Reply:
maybe we can but its not good moment to do this. This change was added because with Bryan we suspected that updater process can be killed after restart and start might be not triggered.